packages/compass/src/main/handle-exception.ts

```diff
@@ -3,12 +3,21 @@ import cleanStack from 'clean-stack';
 import ensureError from 'ensure-error';
 import COMPASS_ICON from './icon';
 
-async function handleException(err: Error): Promise<void> {
+type ExceptionTypes = 'uncaughtException' | 'unhandledRejection';
+
+async function handleException(
+  err: Error,
+  type: ExceptionTypes
+): Promise<void> {
+  // eslint-disable-next-line no-console
+  console.error(`handling ${type}`, err);
```

Comment:
We now log two of the same thing?

Reply:
ah, we are also logging message and detail in error! [0313fd4](https://github.com/mongodb-js/compass/pull/4917/commits/0313fd444918aee529a738efaee73dab877a4a0c)

Reply:
just curious, where would this `console.error` show on Compass when a user is running the app?

Reply:
For the main process, it should just be stdio (e.g. the terminal in which Compass is running, if there is one)